paddle/fluid/framework/ir/graph_test.cc

```diff
@@ -200,9 +200,9 @@ TEST(GraphTest, WriteAfterWrite) {
       ASSERT_TRUE(ir::IsControlDepVar(*n->inputs[1]));
       control_dep2 = n->inputs[1];
       ASSERT_EQ(n->inputs.size(), 2);
-      ASSERT_EQ(control_dep1, control_dep2);
     }
   }
+  ASSERT_EQ(control_dep1, control_dep2);
```

Comment:
also check control_dep1,2 are not nullptr?

Reply:
https://github.com/PaddlePaddle/Paddle/pull/12818/files @panyx0718 , how about this?